guide/english/wordpress/index.md

```diff
@@ -23,19 +23,19 @@ Just to name a few advantages of WordPress:
 * Has mobile app (Android and iOS), which can be used to post and manage their Wordpress webpage.
 * Wordpress sites can be hosted on any server running the LAMP stack
 * WordPress themes make it possible for non-designers to create a customized website
-* Wordpress site are very scalable, you can start as simple blog then end up with complex online stores, membership, and affiliate program
+* Wordpress site is very scalable, you can start an as simple blog then end up with complex online stores, membership, and affiliate program
```

Comment:
1. please revert `are -> is` as the previous iteration is the proper grammar, instead change the word `site` to `sites`
this makes the line `Wordpress sites are very scalable`.

2. remove `an` as it is already correct in previous iteration